spec/system/progress_bar_spec.rb

```diff
@@ -4,80 +4,19 @@
   include_context 'with user'
   include_context 'with progress'
 
-  context 'when on intro section' do
-    describe 'interruption page' do
-      before do
-        visit 'modules/alpha/content-pages/what-to-expect'
-      end
-
-      it 'shows a circle with a green border' do
-        expect(page).to have_css 'li.progress-bar--item:nth-child(1) .fa-circle.green'
-      end
-
-      it 'shows a bold section heading' do
-        expect(page).to have_css 'li.progress-bar--item:nth-child(1) .progress--heading.govuk-\!-font-weight-bold'
-      end
-    end
-
-    describe 'before you start page' do
-      before do
-        visit 'modules/alpha/content-pages/what-to-expect'
-        visit 'modules/alpha/content-pages/before-you-start'
-      end
-
-      it 'shows a circle with a green border' do
-        expect(page).to have_css 'li.progress-bar--item:nth-child(1) .fa-circle.green'
-      end
-
-      it 'shows a bold section heading' do
-        expect(page).to have_css 'li.progress-bar--item:nth-child(1) .progress--heading.govuk-\!-font-weight-bold'
-      end
-    end
-
-    describe 'module intro page' do
-      before do
-        start_module(alpha)
-        visit 'modules/alpha/content-pages/intro'
-      end
-
-      it 'shows a green circle with a tick' do
-        expect(page).to have_selector 'li.progress-bar--item:nth-child(1) .fa-circle-check.green'
-      end
-
-      it 'shows a bold section heading' do
-        expect(page).to have_css 'li.progress-bar--item:nth-child(1) .progress--heading.govuk-\!-font-weight-bold'
-      end
-    end
-  end
-
   context 'when on first submodule' do
```

Comment:
@ebrett I think what's needed here is to move the visual checks along the line and assert them on the second submodule? @agoldstone93 we wouldn't want to delete your checks for green/grey border progression or emboldened headings.

Reply:
specs added to go through three submodules beginning to end